controllers/disruption_rollout_controller.go

```diff
@@ -107,6 +122,7 @@ func (r *DisruptionRolloutReconciler) Reconcile(ctx context.Context, req ctrl.Re
 	}
 
 	if tooLate {
+		r.handleMetricSinkError(r.MetricsSink.MetricTooLate([]string{"rolloutName:" + instance.Name, "namespace:", instance.Namespace, "targetName:", instance.Spec.TargetResource.Name}))
```

Comment:
noticed that we have `[]string{"rolloutName:" + instance.Name, "namespace:", instance.Namespace, "targetName:", instance.Spec.TargetResource.Name}` multiple times. Could we make this a variable or a func to mutualize the code?

Reply:
also let's use disruptionRollout as a suffix.

Reply:
disruptionRollout instead of rolloutName?

Reply:
I meant disruptionRolloutSomethingSomething for all the rollout tags, so here disruptionRolloutName!

Reply:
checkout the comment Philip made below, but I think we can just simplify to "name" because the prefix on the metric already gives us context on whether its rollout or cron.

Reply:
I agree with Claire, I did a bad job explaining myself

Reply:
EDIT: Back to what you suggested 😅